src/Carousel/Carousel.js

```diff
@@ -1,3 +1,32 @@
 import React, { Component } from 'react';
+import data from '../data.js';
+import TypeCard from '../TypeCard/TypeCard.js';
+import '../Styles/Carousel.scss';
 
-//carousel will keep track of what card is selected
\ No newline at end of file
+
+export default class Carousel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
I'm not sure I understand what we would be doing with the state of `visibleCards` here?